packages/core/scripts/serverless-orchestration/ServerlessHub.js

```diff
@@ -150,13 +166,15 @@ hub.post("/", async (req, res) => {
           ? errorOutput.message
           : {
               errorOutputs: Object.keys(errorOutput.errorOutputs), // eslint-disable-line indent
-              validOutputs: Object.keys(errorOutput.validOutputs) // eslint-disable-line indent
+              validOutputs: Object.keys(errorOutput.validOutputs), // eslint-disable-line indent
+              errorMessages // eslint-disable-line indent
             } // eslint-disable-line indent
     });
     await delay(2); // Wait a few seconds to be sure the the winston logs are processed upstream.
     res.status(500).send({
       message: "Some spoke calls returned errors",
-      output: errorOutput instanceof Error ? errorOutput.message : errorOutput
+      output: errorOutput instanceof Error ? errorOutput.message : errorOutput,
+      errorMessages
```

Comment:
Ideally I would add `errorMessages` under `output` like int he `logger.error(...)` above, but I didn't want to mess up the shape of the `output` key in this server response because I do not know what is downstream of this.

Also, curious if this should be renamed to `spokeErrorMessages` for example, since only errors originating from the spokes will appear here. If the error comes from the Hub code, then there will not be any `errorMessages`